test/csv/interface/test_write.rb

```diff
@@ -78,6 +78,15 @@ def test_generate_line_row_sep
     LINE
   end
 
+  def test_generate_lines
```

Comment:
Could you move this under `#test_generate_line_shortcut`?
I want to keep `test_generate_line*` grouping.

Reply:
Sure!